src/cs4450_project/FPCameraController.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * file: FPCameraController.java
+ * author: Isaac Quintanilla
+ * class: CS 4450 - Computer Graphics
+ *
+ * assignment: Project
+ * date last modified: 10/9/2021
+ *
+ * purpose: This class will hold our camera's position in 3D space.
+ */
+package cs4450_project;
+
+import org.lwjgl.util.vector.Vector3f;
+import org.lwjgl.input.Keyboard;
+import org.lwjgl.input.Mouse;
+import org.lwjgl.opengl.Display;
+import static org.lwjgl.opengl.GL11.*;
+import org.lwjgl.Sys;
+
+public class FPCameraController {
+    // include CUBE class for gameloop render
+    Cube cube = new Cube(0, 0, 0, 1);
+    // 3D vector to store the camera's position
+    private Vector3f position = null;
+    private Vector3f lPosition = null;
+    
+    // the rotation around the Y axis of the camera
+    private float yaw = 0.0f;
+    
+    // the rotation around the X axis of the camera
+    private float pitch = 0.0f;
+    
+    private Vector3Float me;
+    
+    public FPCameraController(float x, float y, float z){
+	// instantiate position Vector3f to the x y z parameters
+	position = new Vector3f(x, y, z);
+	lPosition = new Vector3f(x, y, z);
+	lPosition.x =0f;
+	lPosition.y = 15f;
+	lPosition.z = 0f;
+	
+    }
+    
+    // increment the camera's current yaw rotation
+    public void yaw(float amount){
+	// increment the yaw by the amount parameter
+	yaw+= amount;
```

Comment:
```suggestion
	yaw += amount;
```
Internal comments unneeded.